src/py21cmfast/src/HaloField.c

```diff
@@ -84,15 +84,17 @@ int ComputeHaloField(float redshift_desc, float redshift, InitialConditions *box
         // store highly used parameters
         int grid_dim = simulation_options_global->DIM;
         int z_dim = D_PARA;
+        int dim_arr[3] = {grid_dim, grid_dim, z_dim};
```

Comment:
sorry to be persnickety, but could we call this variable the same as it is in other c modules? I think there it's called `box_dim`